docs/modules/workflow/proc-ploigos-workflow-reference-impl.adoc

```diff
@@ -2,4 +2,175 @@
 
 = Reference Implementation
 
-TODO
+The following guide shows how to deploy the reference implementation of the Ploigos Standard Workflow into an OCP 4.x cluster with a simple quarkus application as a demo. 
+
+
+== Prerequisites
+
+* Operational OCP 4.x cluster
+
+
+== Installing the Ploigos Software Factory Operator
+
+See https://github.com/ploigos/ploigos-software-factory-operator[ploigos-software-factory-operator] for latest information.
+
+
+. Login to an OCP cluster as `cluster-admin`
+. Import the RedHatGov Operator Catalog (Create a custom `CatalogSource`)
++
+----
+oc apply -f - << EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: redhatgov-operators
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: quay.io/redhatgov/operator-catalog:latest
+  displayName: Red Hat NAPS Community Operators
+  publisher: RedHatGov
+EOF
+----
++
+. Create a project named *devsecops* for the pipeline tooling+
++
+----
+oc new-project devsecops
+----
++
+. (Optional) Remove too-restrictive limit ranges *NOTE: Required for RHPDS clusters*
```

Comment:
This won't break if the`LimitRange` doesn't exist - we should probably just make this step required

Reply:
I'll change this, but we should collect some info on what valid limits are for TSSC. Unlimited seems too open.